core/src/main/java/org/apache/carbondata/core/scan/collector/impl/DictionaryBasedVectorResultCollector.java

```diff
@@ -51,8 +51,8 @@
 
   public DictionaryBasedVectorResultCollector(BlockExecutionInfo blockExecutionInfos) {
     super(blockExecutionInfos);
-    queryDimensions = tableBlockExecutionInfos.getQueryDimensions();
-    queryMeasures = tableBlockExecutionInfos.getQueryMeasures();
+    queryDimensions = tableBlockExecutionInfos.getActualQueryDimensions();
```

Comment:
Why getting `actualQueryDimensions` instead of `queryDimensions` in `DictionaryBasedVectorResultCollector`? But we get only `queryDimensions` in `DictionaryBasedResultCollector`